design/craft/inclusivecache/src/Directory.scala

```diff
@@ -132,7 +132,7 @@ class Directory(params: InclusiveCacheParameters) extends Module
   val hit = hits.orR
 
   io.result.valid := ren2
-  io.result.bits := Mux(hit, Mux1H(hits, ways), Mux(setQuash && (tagMatch || wayMatch), bypass.data, Mux1H(victimWayOH, ways)))
+  (io.result.bits: Data).waiveAll :<>= (Mux(hit, Mux1H(hits, ways), Mux(setQuash && (tagMatch || wayMatch), bypass.data, Mux1H(victimWayOH, ways))): Data).waiveAll
```

Comment:
```suggestion
  io.result.bits.viewAsSupertype(chiselTypeOf(bypass.data)) :<>= Mux(hit, Mux1H(hits, ways), Mux(setQuash && (tagMatch || wayMatch), bypass.data, Mux1H(victimWayOH, ways)))
```

Reply:
Please use `DataView` API to for subclass connectivity.

Reply:
FYI: https://github.com/chipsalliance/chisel/blob/068eb8f789580e04f95b016151ea449466d67e61/docs/src/explanations/connectable.md?plain=1#L572-L593